cmd/server/customers.go

```diff
@@ -170,27 +250,46 @@ type customerStatusRequest struct {
 	Status string `json:"status"` // TODO(adam): better name for Default ?
 }
 
-func updateCustomerStatus(logger log.Logger, searcher *searcher) http.HandlerFunc {
+func updateCustomerStatus(logger log.Logger, searcher *searcher, custRepo customerRepository) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		w = wrapResponseWriter(logger, w, r)
 
+		custId := getCustomerId(w, r)
+		userId := moovhttp.GetUserId(r)
+		if userId == "" {
+			moovhttp.Problem(w, errNoUserId)
+			return
+		}
+
 		var req customerStatusRequest
 		if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
 			moovhttp.Problem(w, err)
 			return
 		}
+
 		req.Status = strings.ToLower(strings.TrimSpace(req.Status))
-		switch req.Status {
-		case "blocked":
-			// update customer status to blocked
-		default:
-			if req.Status == "" {
-				moovhttp.Problem(w, errors.New("no status provided"))
+		if req.Status == "blocked" {
```

Comment:
Actually, this should add an unblock record, so we can have a way to whitelist a customer.